static/app/actionCreators/modal.tsx

```diff
@@ -277,9 +277,16 @@ export async function openDashboardWidgetLibraryModal(
   openModal(deps => <Modal {...deps} {...options} />, {backdrop: 'static', modalCss});
 }
 
-export async function openWidgetViewerModal(options: WidgetViewerModalOptions) {
+export async function openWidgetViewerModal(
+  onClose,
+  options: WidgetViewerModalOptions & {onClose?: () => void}
```

Comment:
do we need two `onClose`?

can we add a type for the first parameter?

Reply:
oops, definitely only need one `onClose`. Updated to just the first param and added type!